### PR TITLE
UserSearch Component Modification

### PR DESCRIPTION
**Pull request by nanyangpro**
# Description

This PR adds the user not exist story to the UserSearch component and fixes the font defect.

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

The corresponding Storybook story has been created to confirm that the user search component looks and behaves as it should.

Code style and potential errors/issues have been taken care of by ESLint and Prettier.

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: maltd-frontend/src/components/composite/UserSearch/UserSearch.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ export default function UserSearch({
         projects.
       </p>
       <p />
-      {!state.isLoading && (
+      {!state.isLoading && state.userExists && (
```
Comment (akroon3r) [inline maltd-frontend/src/components/composite/UserSearch/UserSearch.js]: the trailing && is a little confusing, is this a react thing I'm not aware of? sorry for my ignorance

File: maltd-frontend/src/components/composite/UserSearch/UserSearch.js [hunk 2]
```diff
@@ -44,6 +44,26 @@ export default function UserSearch({
...
+      {!state.userExists && (
+        <div>
+          <div>
+            <InputField inputField={inputField} />
+            <p />
+            <div>
+              <div className="float-left">
+                <p className="float-left">
+                  <small>
+                    The user does not exist, please retype your IDIR username...
```
Comment (akroon3r) [inline maltd-frontend/src/components/composite/UserSearch/UserSearch.js]: "This user does not exist, please retype **the** IDIR username."

Thoughts?

File: maltd-frontend/src/components/composite/UserSearch/UserSearch.js (hunk 1 above)
Comment (sdevalapurkar) [reply maltd-frontend/src/components/composite/UserSearch/UserSearch.js]: yeah it is a way to add conditional statements in react, this is fine

File: maltd-frontend/src/components/composite/UserSearch/UserSearch.stories.js [hunk 3]
```diff
@@ -5,11 +5,15 @@ import "bootstrap/dist/css/bootstrap.css";
 import UserSearch from "./UserSearch";
 
 const userSearch = {
-  state: { isLoading: false }
+  state: { isLoading: false, userExists: true }
```
Comment (sdevalapurkar) [inline maltd-frontend/src/components/composite/UserSearch/UserSearch.stories.js]: `userExists` should have 3 possible states which are
1) null -> meaning the user has not entered anything yet and hasnt made a request with an idir -> this is the default state
2) true -> meaning the user has entered and searched based on an idir that is valid
3) false -> meaning the user has entered and searched based on an idir that is invalid

File: maltd-frontend/src/components/composite/UserSearch/UserSearch.stories.js [hunk 4]
```diff
@@ -5,11 +5,15 @@ import "bootstrap/dist/css/bootstrap.css";
...
 };
 
 const isLoadingUserSearch = {
-  state: { isLoading: true }
+  state: { isLoading: true, userExists: true }
+};
+
+const userNotExist = {
```
Comment (sdevalapurkar) [inline maltd-frontend/src/components/composite/UserSearch/UserSearch.stories.js]: call this `noUserExists`

Comment (sdevalapurkar) [reply maltd-frontend/src/components/composite/UserSearch/UserSearch.stories.js]: we dont need an entire object for this... this should be cleaned up in terms of naming

File: maltd-frontend/src/components/composite/UserSearch/UserSearch.stories.js [hunk 5]
```diff
@@ -56,6 +60,17 @@ storiesOf("UserSearch", module)
       generalButton={generalButton}
     />
   ))
+  .add("not exist", () => (
```
Comment (sdevalapurkar) [inline maltd-frontend/src/components/composite/UserSearch/UserSearch.stories.js]: nit: `no user exists`

**Pull request merged**